components/Card.tsx

```diff
@@ -4,34 +4,50 @@ import { CldImage } from "next-cloudinary";
 type CardProps = {
   name: string;
   cloudinarySrc: string;
+  languages: string[];
+  gender: string;
+  skill: string;
 };
 
-export default function Card({ name, cloudinarySrc }: CardProps): JSX.Element {
+export default function Card({
+  name,
+  cloudinarySrc,
+  languages,
+  gender,
+  skill,
+}: CardProps): JSX.Element {
   return (
     <StyledDiv>
       <CldImage
         width="80"
         height="80"
         src={cloudinarySrc}
-        alt="a profile image"
+        alt={`the profile image of ${name}`}
       />
       <p>{name}</p>
+      <p>{skill}</p>
+      <p>{gender}</p>
+
+      {languages.map((language: string) => {
+        return <p key={language}>{language}</p>;
```

Comment:
if this con be more than one, I suggest using a `list` to list all your languages.

Reply:
👍

Reply:
I don´t know why it looked black in the page of the new users. It works normally in my laptop 🤷🏽